Funky-Chickens/workFORCEmanagement
# Cait emps to training detail

added a new computer form page and functionality to add a computer. added functionality that pops up a confirmation to delete computers and delete button only appears if it has never been assigned. updated bootstrap to v4 and added popper library.

to test
run "npm install"
navigate to the computer list page, click add new button
fill out form and submit. view list to see that computer was added

navigate to a computer that has been assigned, see that the delete button is not there
navigate to a computer that has not been assigned and see that the delete button is there
click the delete button and see that a confirmation message appears. click the yes button and see that the computer has been deleted from the list
